pkgs/development/python-modules/neurokit2/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  pytest_7,
+  scipy,
+  scikit-learn,
+  pandas,
+  matplotlib,
+  requests,
+  cvxopt,
+  biosppy,
+}:
+
+buildPythonPackage rec {
+  pname = "neurokit2";
+  version = "0.2.10";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "neuropsychology";
+    repo = "NeuroKit";
+    tag = "v${version}";
+    hash = "sha256-e/B1JvO6uYZ6iVskFvxZLSSXi0cPep9bBZ0JXZTVS28=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail '"pytest-runner"' '"pytest"'
+  '';
+
+  build-system = [
+    setuptools
+    pytest_7
+  ];
+
+  dependencies = [
+    scipy
+    scikit-learn
+    pandas
+    matplotlib
+    requests
+    cvxopt
+    biosppy
+  ];
+
+  # Passing tests do not meet coverage criteria
```

Comment:
Note that Python packages shouldn't run `pytest-cov`. Not sure how it's getting into the closure here, but best to patch it out by adding `pytest-cov-stubs` to `nativeCheckInputs`. In that case, it could be possible to enable the tests.